README.md

```diff
@@ -48,6 +48,27 @@ script/server --mock-api
 
 API endpoint stubbing is set up in `/wiremock/scripts/stubApis.ts`.
 
+
+### Local user accounts
+
+There are three user accounts with different roles that can be used when running the application locally:
```

Comment:
I wonder if it's worth saying why you'd use one over the other? Maybe this can come later when we use these roles, but it would be good to know what different functionality the different roles would provide